browser/about_flags.cc

```diff
@@ -104,6 +103,10 @@
 #include "brave/components/psst/common/features.h"
 #endif
 
+inline constexpr char kPlaylistFeatureInternalName[] = "playlist";
+inline constexpr char kPlaylistFakeUAFeatureInternalName[] = "playlist-fake-ua";
+inline constexpr char kSplitViewFeatureInternalName[] = "brave-split-view";
+
```

Comment:
Should these go into the anonymous namespace below (line 130)?

Reply:
this file is a bit of a mess of overrides, etc... so maybe I missed something? I didn't even see any other InternalName constants even though upstream has them

Reply:
upstream just puts these inline with everything else, but I don't think they even need constants and could just go inline like the others

Reply:
I triple checked to make sure the values didn't get mixed up somehow